cms/analysis/models.py

```diff
@@ -71,17 +73,19 @@ def previous_releases(self, request: "HttpRequest") -> "TemplateResponse":
             request,
             # TODO: update to include drafts when looking at previews holistically.
             context_overrides={"pages": AnalysisPage.objects.live().child_of(self).order_by("-release_date")},
-            template="templates/pages/analysis_page--previous-releases.html",
+            template="templates/pages/statistical_article--previous-releases.html",
```

Comment:
I think this also doesn't match the template, should be `"templates/pages/statistical_article_page--previous-releases.html"`?

Reply:
same as https://github.com/ONSdigital/dis-wagtail/pull/69#discussion_r1908890581